dev/list_input.sh

```diff
@@ -136,6 +135,8 @@ list_input() {
   unset _list_options
   unset _break_keypress
   unset _first_keystroke
+
+  cleanup
```

Comment:
Thanks for handling the cleanup here (and others)! I initially only handled it during script exit, then I realized it might not be enough if a user wants to do something other stuff during the execution of the script.